SimplaeWAWS.Models/Site.cs

```diff
@@ -0,0 +1,177 @@
+        public string SiteName { get; private set; }
+
+        public Dictionary<string, string> AppSettings { get; set; }
+
+        public Dictionary<string, string> Metadata { get; set; }
+
+        public string HostName { get; set; }
+
+        public string ScmHostName { get; set; }
+
+        public Site(string subscriptionId, string resourceGroupName, string name, string kind = null)
+            : base (subscriptionId, resourceGroupName)
+        {
+            this.SiteName = name;
+            this.Kind = kind;
+        }
+
+        public string Url
+        {
+            get 
+            {
+                return String.Format(CultureInfo.InvariantCulture, "https://{0}/", HostName);
+            }
+        }
+
+        public string GetMobileUrl(string templateName)
+        {
+            return Url + (templateName.Equals("Todo List", StringComparison.OrdinalIgnoreCase) ? "jsclient" : "admin");
+        }
+
+        public string ScmUrl
+        {
+            get
+            {
+                return String.Format(CultureInfo.InvariantCulture, "https://{0}/", ScmHostName);
+            }
+        }
+
+        public string ScmUrlWithCreds
+        {
+            get
+            {
+                return String.Format(CultureInfo.InvariantCulture, "https://{0}:{1}@{2}/", PublishingUserName, PublishingPassword, ScmHostName);
+            }
+        }
+
+        public string KuduConsoleWithCreds
+        {
+            get
+            {
+                return ScmUrlWithCreds + "DebugConsole";
+            }
+        }
+
+        public string GitUrlWithCreds
+        {
+            get
+            {
+                return ScmUrlWithCreds + SiteName + ".git";
+            }
+        }
+
+        public string MonacoUrl
+        {
+            get
+            {
+                return ScmUrl + "dev";
+            }
+        }
+
+        public string ContentDownloadUrl
+        {
+            get
+            {
+                return ScmUrl + "zip/site/wwwroot";
+            }
+        }
+
+        public string IbizaUrl
+        {
+            get
+            {
+                return string.Concat("https://portal.azure.com/", SimpleSettings.TryTenantName, "#resource", CsmId);
+            }
+        }
+
+        public string PublishingUserName { get; set; }
+
+        public string PublishingPassword { get; set; }
+
+        public bool IsSimpleWAWSOriginalSite
+        {
+            get
+            {
+                return !string.IsNullOrEmpty(SiteName) &&
+                       Regex.IsMatch(SiteName, "^[A-F0-9]{8}-0ee0-4-231-b9ee$", RegexOptions.IgnoreCase);
+            }
+        }
+        public string Kind { get; set; }
+
+        public bool IsFunctionsContainer
+        {
+            get
+            {
+
+                return !string.IsNullOrEmpty(Kind) &&
+                    Kind.StartsWith(Constants.FunctionsContainerSiteKind);
+            }
+        }
+
+        public bool NameStartsWithFunctions
```

Comment:
Aah its a duplicate file. The directory  doesnt need to be there. its in the wrong directory . I'll remvoe the file